packages/material-ui/src/Dialog/Dialog.js

```diff
@@ -189,8 +200,13 @@ class Dialog extends React.Component {
           {...TransitionProps}
         >
           <div
-            className={classNames(classes.container, classes[`scroll${capitalize(scroll)}`])}
-            onClick={this.handleBackdropClick}
+            className={classNames(
+              'mui-fixed',
```

Comment:
For what is that one used?

Reply:
I moved it from Modal to fix:
![image](https://user-images.githubusercontent.com/12938082/49542223-e1327f80-f8cc-11e8-8365-d6c3c05973d1.png)
occurring when a Dialog has scroll='body' set. I still need to confirm with @oliviertassinari that this is correct

Reply:
Just wondering if this is a public class, for docs or testing only.

Reply:
Ah, it was added in #13674
https://material-ui.com/getting-started/faq/#why-do-the-fixed-positioned-elements-move-when-a-modal-is-opened